scripts/nbconverted/plot_temporal_geneExp.py

```diff
@@ -0,0 +1,132 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee (August 2018) 
+#
+# Plot heatmap of gene expression data as environment change from high to low oxygen levels
+#
+# Dataset: Pseudomonas aeruginosa gene expression compendium referenced in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069748/
+# 
+# Use map_file to select only those samples from the oxygen level experiment
+# 
+#-------------------------------------------------------------------------------------------------------------------------------
+import os
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from scipy.stats import pearsonr
+
+
+randomState = 123
+from numpy.random import seed
+seed(randomState)
+
+
+# In[3]:
+
+
+# load arguments
+data_file = os.path.join(os.path.dirname(os.getcwd()), "data", "all-pseudomonas-gene-normalized.zip")  # repo file is zipped
+map_file = os.path.join(os.path.dirname(os.getcwd()), "metadata", "mapping_oxy.txt")
+
+PA1673like_file = os.path.join(os.path.dirname(os.getcwd()), "output", "PA1673_like_genes_test.txt")
+
+
+# In[4]:
+
+
+# read in data
+data = pd.read_table(data_file, header = 0, sep = '\t', index_col = 0, compression='zip')
```

Comment:
no `=` arg spaces - would recommend a targeted find and replace strategy

Reply:
fixed all